Classes/EventPublisher/EventPublisherInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+namespace Neos\EventSourcing\EventPublisher;
+
+/*
+ * This file is part of the Neos.EventSourcing package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\EventSourcing\Event\DomainEvents;
+
+/**
+ * Contract for an Event Publisher that is invoked by the Event Store when ever new events where committed
```

Comment:
WHAT is an Event Publisher?

-> is used to update projections whenever new events are stored
-> also other event listeneres are invoked from here.

Reply:
Good call, I'll add some description here!